cipherscan

```diff
@@ -31,6 +31,7 @@ else
     case "$(uname -s)" in
         Darwin)
             opensslbin_name="openssl-darwin64"
+            openssl_legacy_bin_name="openssl-darwin64"
```

Comment:
that's the same file?

Reply:
Yeah, until I build a recent openssl for darwin.